pkg/exporter/sumologicexporter/exporter.go

```diff
@@ -386,6 +343,10 @@ func (se *sumologicexporter) configure(ctx context.Context) error {
 		foundSumoExt bool
 	)
 
+	if se.config.CompressEncoding != NoCompression {
```

Comment:
The default for compress_encoding is `gzip`, so this will happen if the user doesn't set anything, which I don't think is what we want? We should check for the default here.

Reply:
actually `compress_encoding` isn't set by default anymore. Instead, `Compression` is now set by default to be `gzip` here - https://github.com/SumoLogic/sumologic-otel-collector/blob/zstd-support/pkg/exporter/sumologicexporter/config.go#L103

Reply:
so, the default is now driven by `Compression` and if `CompressEncoding` is set it takes precedence over `Compression`.

Reply:
@swiatekm-sumo does this make sense?, I think the current condition is valid, because `CompressEncoding` isn't set by default anymore.

Reply:
Ok, makes sense.